awx/api/serializers.py

```diff
@@ -1409,11 +1409,11 @@ def validate(self, attrs):
 
 class ExecutionEnvironmentSerializer(BaseSerializer):
     show_capabilities = ['edit', 'delete', 'copy']
-    managed_by_tower = serializers.ReadOnlyField()
+    managed_by_controller = serializers.ReadOnlyField()
```

Comment:
From a practical perspective, this would need to merge after @jbradberry's changes for EEs.